charts/piraeus/values.yaml

```diff
@@ -26,9 +26,14 @@ operator:
     dbCertSecret: ""
     dbUseClientCert: false
     sslSecret: ""
+    linstorHttpsControllerSecret: ""
+    linstorHttpsClientSecret: ""
+    linstorHttpsUseClientCert: false
   nodeSet:
     drbdKernelModuleInjectionMode: Compile
     satelliteImage: "quay.io/piraeusdatastore/piraeus-server:v1.7.0"
     kernelModImage: "quay.io/piraeusdatastore/drbd9-bionic:v9.0.22"
     storagePools: null
     sslSecret: ""
+    linstorHttpsUseClientCert: false
+    linstorHttpsClientSecret: ""
```

Comment:
I don't think it makes much sense to duplicate these keys in the Helm values. Either we just use the ones specified in `controllerSet` everywhere, or we put them in their own section.

Reply:
Moved to the top level